timetracker/vms/models.py

```diff
@@ -309,10 +309,20 @@ def __str__(self):
             A string describing the time record including the start and
             end times.
         """
+        st = self.time_start
```

Comment:
I think it's clearer to use the full attribute name (ie `self.time_start`) rather than an abbreviation, but you can go either way here.